.errors/errors_kafka.go

```diff
@@ -0,0 +1,624 @@
+package errors
```

Comment:
What is the purpose of this file? I don't think it is needed here.

Reply:
This is initial file that was changed. Initially PR will be created with just that file for review.
If change has no error code changes then we close PR etc.

Reply:
I still do not understand what it is for and how it would be consumed. It looks like the errors file from the kas-fleet-manager codebase..

Reply:
Anyway, the approach is proven to work, so we need not worry about this detail yet.